README.md

```diff
@@ -67,6 +67,9 @@ To try the package, simply install it (in a virtualenv or globally) and run `fla
     # On macOS, install with Homebrew:
     brew install andreoliwa/formulae/nitpick
 
+    # On archlinux, install with yay:
+    yay -Syu nitpick
+
```

Comment:
Can you also add this to the RST file?
https://github.com/andreoliwa/nitpick/blob/develop/docs/quickstart.rst#install--basic-usage